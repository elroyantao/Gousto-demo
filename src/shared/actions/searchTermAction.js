
export const setSearchTerm = (searchTerm) => ({
  type: 'SET_SEARCH_TERM',
  searchTerm
})

export const clearSearchTerm = () => ({
  type: 'CLEAR_SEARCH_TERM'
})
