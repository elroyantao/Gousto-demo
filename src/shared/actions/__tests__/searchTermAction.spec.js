import * as actions from '../searchTermAction'

describe('searchTerm actions', () => {
  it('setSearchTerm()', () => {
    expect(actions.setSearchTerm('fake search term')).toMatchSnapshot()
  })
  it('clearSearchTerm()', () => {
    expect(actions.clearSearchTerm()).toMatchSnapshot()
  })
})
