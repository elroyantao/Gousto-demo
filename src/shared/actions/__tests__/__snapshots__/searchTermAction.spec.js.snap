// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchTerm actions clearSearchTerm() 1`] = `
Object {
  "type": "CLEAR_SEARCH_TERM",
}
`;

exports[`searchTerm actions setSearchTerm() 1`] = `
Object {
  "searchTerm": "fake search term",
  "type": "SET_SEARCH_TERM",
}
`;
